pom.xml

```diff
@@ -88,7 +88,7 @@
     <dependency>
       <groupId>org.fusesource.jansi</groupId>
       <artifactId>jansi</artifactId>
-      <version>1.12</version>
+      <version>1.14-SNAPSHOT</version>
```

Comment:
I don't think we want to ship with an unshipped version of 1.14. I am not saying we shouldn't merge this, but we should be careful releasing with this.